### ISSUE TITLE
Loss is NaN on low LR

### ISSUE THREAD

**Issue by recoilme** [NVlabs/Sana#65]
Hello, im trying to finetune SANA

All looks good at start, but then i try to learn details on low LR i get Nan on lr=6e-6, 7e-6 and from time to time on 8e-6
6e-6 - https://wandb.ai/recoilme/sanaft/runs/20?nw=nwuserrecoilme
7e-6 - https://wandb.ai/recoilme/sanaft/runs/19?nw=nwuserrecoilme

So i dont may get SDXL details level quality, all little blurry
![validation_1_f34ef3ab0dc7f726c9d5](https://github.com/user-attachments/assets/de0c4721-b1cb-4540-81af-2872a7a4393b)

I see similar problem in original Sana.. How we may to resolve this?

**Comment by lawrence-cj** [comment]
I may caused by non-stable of FP16, we will release a BF16 version later for fine-tuning.
